net-im/discord-bin/discord-bin-0.0.8.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2019 Gentoo Foundation
```

Comment:
Old copyright header, please use repoman to commit to fix it for you automatically.

Reply:
Done